modules/PlayersJsReceiver/PlayersJsReceiver.loader.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Receiver loader
+ * 
+ * Uses mw.PluginManager.add and kalturaPluginName, no loader needed
+ */
```

Comment:
@mdale - do we need to add an empty loader if we have no preinitialisation logic?